packages/eslint-config-sheriff/eslint.config.js

```diff
@@ -0,0 +1,30 @@
+import { sheriff } from 'eslint-config-sheriff';
+import { defineFlatConfig } from 'eslint-define-config';
+
+const sheriffOptions = {
+  react: false,
+  next: false,
+  lodash: true,
+  remeda: false,
+  astro: false,
+  playwright: false,
+  jest: false,
+  vitest: false,
+};
+
+export default defineFlatConfig([
+  ...sheriff(sheriffOptions),
+  {
+    rules: {
+      'fsecond/prefer-destructured-optionals': 0,
+      '@typescript-eslint/no-unsafe-return': 0,
+      '@typescript-eslint/no-unsafe-member-access': 0,
+      '@typescript-eslint/no-unsafe-argument': 0,
+      'sonarjs/no-duplicate-string': 0,
+      'lodash-f/import-scope': 0,
+    },
+  },
+  {
+    ignores: ['tsup.config.js'],
+  },
```

Comment:
not a bug. 

`const supportedFileTypes: "**/*{js,mjs,cjs,ts,mts,cts,jsx,tsx,mtsx,mjsx,astro}"`

Reply:
We support js!?
Is that new? Do the docs need updating?

Reply:
No, i mean, it has always been like that. In the docs we basically _discourage_ Sheriff usage in pure Vanilla JS projects, but i believe you can kinda make it work if you want. The config is meant to be used in TS projects, but if you enable/disable some rules it work in JS projects too just fine, i believe.

Reply:
Wait, so Sheriff supports JS, it just doesn't support _pure_ JS (i.e., no `tsconfig.json` file)?

Reply:
Like i said in previous comment, i want to discourage people from using Sheriff in pure JS codebases because that's not what Sheriff is made for.

But i didn't test Sheriff too much in pure JS codebases, so i'm not too sure what would happen.

Reply:
I might just test it out and report back :)
I certainly prefer TS syntax, but I know some people like their JSDocs, and Sheriff should be able to Just Work™️ for those codebases as well.

Reply:
> Sheriff should be able to Just Work™️ for those codebases as well

Well, it's stated pretty clearly in the docs that Sheriff is not meant to be used outside of TS codebases. If someone decides to do it anyway, he is on his own.